neo4j-ubuntu-vm/azuredeploy.json

```diff
@@ -0,0 +1,433 @@
+        "Standard_D64_v3",
+        "Standard_D2s_v3",
+        "Standard_D4s_v3",
+        "Standard_D8s_v3",
+        "Standard_D16s_v3",
+        "Standard_D32s_v3",
+        "Standard_D64s_v3",
+        "Standard_E2_v3",
+        "Standard_E4_v3",
+        "Standard_E8_v3",
+        "Standard_E16_v3",
+        "Standard_E32_v3",
+        "Standard_E64_v3",
+        "Standard_E2s_v3",
+        "Standard_E4s_v3",
+        "Standard_E8s_v3",
+        "Standard_E16s_v3",
+        "Standard_E32s_v3",
+        "Standard_E64s_v3"
+      ],
+      "metadata": {
+        "description": "Virtual machine size. Make sure VM size supports disk type."
+      }
+    },
+    "vmUbuntuOSVersion": {
+      "type": "string",
+      "defaultValue": "17.10",
+      "allowedValues": [
+        "17.04",
+        "17.10"
+      ],
+      "metadata": {
+        "description": "Ubuntu OS image version."
+      }
+    },
+    "storageaccountPrefix": {
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "Storage account name prefix."
+      }
+    },
+    "storageaccountType": {
+      "type": "string",
+      "defaultValue": "Premium_LRS",
+      "allowedValues": [
+        "Standard_LRS",
+        "Standard_ZRS",
+        "Standard_GRS",
+        "Standard_RAGRS",
+        "Premium_LRS"
+      ],
+      "metadata": {
+        "description": "Storage account type. Premium storage is recommended for graph databases. Make sure storage type is supporting by VM size."
+      }
+    },
+    "vmAdminUsername": {
+      "type": "string",
+      "defaultValue": "azureuser",
+      "minLength": 1,
+      "metadata": {
+        "description": "User name for the virtual machine."
+      }
+    },
+    "sshKeyData": {
+      "type": "string",
+      "metadata": {
+        "description": "SSH rsa public key file as a string."
+      }
+    },
+    "neo4jEdition": {
+      "type": "string",
+      "defaultValue": "neo4j",
+      "allowedValues": [
+        "neo4j",
+        "neo4j-enterprise",
+        "neo4j=3.2.1",
+        "neo4j-enterprise=3.2.1",
+        "neo4j=3.2.5",
+        "neo4j-enterprise=3.2.5",
+        "neo4j=3.3.0",
+        "neo4j-enterprise=3.3.0"
+      ],
+      "metadata": {
+        "description": "Neo4J edition and version to install."
+      }
+    },
+    "publicIPAddressDns": {
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "Unique DNS name for public IP Address. As in: publicIPAddressDns.location.cloudapp.azure.com"
+      }
+    },
+    "_artifactsLocation": {
+      "type": "string",
+      "metadata": {
+        "description": "The base URI where artifacts required by this template are located. When the template is deployed using the accompanying scripts, a private location in the subscription will be used and this value will be automatically generated."
+      },
+      "defaultValue": "https://raw.githubusercontent.com/helshabini/neo4j-ubuntu-vm/master/"
```

Comment:
this needs to point to Azure master

Reply:
Since this is the first pull, the script does not exist on the Azure master branch yet. When I tried this Travis CI generated an error as it couldn't reach the script.
I had to point to a different location temporarily and I will fix it on the next pull, unless you have another suggestion.